pypesto/objective.py

```diff
@@ -17,8 +17,12 @@
 except ImportError:
     amici = None
     pass
-    # we only pass import errors, if the initialization of the module itself
-    # fails this should not be caught right?
+
+try:
+    import pandas as pd
```

Comment:
I would rather put pandas in setup.py -> install_requires (might have missed that), and avoid try here, since it will be needed in many places anyway